File: app/src/main/java/com/chuongvd/app/example/ui/auth/login/LoginActivity.java
package com.chuongvd.app.example.ui.auth.login;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import com.chuongvd.app.example.R;

public class LoginActivity extends AppCompatActivity {

    public static Intent newIntance(Context context) {
        Intent intent = new Intent(context, LoginActivity.class);
        return intent;
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);
//        try {
//            PackageInfo info = getPackageManager().getPackageInfo(BuildConfig.APPLICATION_ID,
//                    PackageManager.GET_SIGNATURES);
//            for (Signature signature : info.signatures) {
//                MessageDigest md = MessageDigest.getInstance("SHA");
//                md.update(signature.toByteArray());
//                Log.d("KeyHash:", Base64.encodeToString(md.digest(), Base64.DEFAULT));
//            }
//        } catch (PackageManager.NameNotFoundException e) {
//
//        } catch (NoSuchAlgorithmException e) {
//
//        }
        if (savedInstanceState == null) {
            LoginFragment fragment = new LoginFragment();
            getSupportFragmentManager().beginTransaction()
                    .add(R.id.fragment_container, fragment, LoginFragment.class.getName())
                    .commit();
        }
    }
}

File: app/src/main/java/com/chuongvd/app/signal/ui/auth/login/LoginActivity.kt
package com.chuongvd.app.signal.ui.auth.login

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.chuongvd.app.signal.R

class LoginActivity : AppCompatActivity() {

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_login)
    //        try {
    //            PackageInfo info = getPackageManager().getPackageInfo(BuildConfig.APPLICATION_ID,
    //                    PackageManager.GET_SIGNATURES);
    //            for (Signature signature : info.signatures) {
    //                MessageDigest md = MessageDigest.getInstance("SHA");
    //                md.update(signature.toByteArray());
    //                Log.d("KeyHash:", Base64.encodeToString(md.digest(), Base64.DEFAULT));
    //            }
    //        } catch (PackageManager.NameNotFoundException e) {
    //
    //        } catch (NoSuchAlgorithmException e) {
    //
    //        }
    if (savedInstanceState == null) {
      val fragment = LoginFragment()
      supportFragmentManager.beginTransaction()
          .add(R.id.fragment_container, fragment, LoginFragment::class.java.name)
          .commit()
    }
  }

  companion object {

    fun newIntance(context: Context): Intent {
      return Intent(context, LoginActivity::class.java)
    }
  }
}
